SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.cpp

```diff
@@ -22,75 +23,146 @@ int32 UGenerateSchemaAndSnapshotsCommandlet::Main(const FString& Args)
 {
 	UE_LOG(LogSpatialGDKEditorCommandlet, Display, TEXT("Schema & Snapshot Generation Commandlet Started"));
 
-	//NOTE: For future use, if schema generation configuration at the command line is desired
-	//TArray<FString> Tokens;
-	//TArray<FString> Switches;
-	//TMap<FString, FString> Params;
-	//ParseCommandLine(*Args, Tokens, Switches, Params);
+	TArray<FString> Tokens;
+	TArray<FString> Switches;
+	TMap<FString, FString> Params;
+	ParseCommandLine(*Args, Tokens, Switches, Params);
+
+	//TODO: Optionally clean up previous schema and snapshot files once the UnrealGDK provides support to do so
```

Comment:
did you put in a feature request for this issue?

Reply:
Yes: UNR-954

Reply:
Could you please link the ticket number in the comment.